resources-generator/src/main/kotlin/dev/icerock/gradle/utils/StringExt.kt

```diff
@@ -42,11 +41,7 @@ internal val String.flatName: String
     get() = this.remove('.')
 
 internal fun String.convertXmlStringToLocalizationValue(): String {
-    return StringEscapeUtils.unescapeXml(this).let {
-        UnicodeUnescaper().translate(
-            StringEscapeUtils.escapeJava(it)
-        )
-    }
+    return StringEscapeUtils.unescapeXml(this)
```

Comment:
Are you sure that it works correctly?
Bug was about iOS.

Add please cases with " ' % @ * {} symbols too

Reply:
i'm add sample text with symbols, for " and ' need mirroring for correct work on all platforms (exclude js: that platform incorrect work with mirroring, my opinion)